libFDK/include/FDK_archdef.h

```diff
@@ -240,6 +240,38 @@ amm-info@iis.fraunhofer.de
 #define LDCOEFF_16BIT
 #define WINDOWTABLE_16BIT
 
+#elif defined(__sparc__)
+#define ARCH_PREFER_MULT_32x32
+#define ARCH_PREFER_MULT_32x16
+#define SINETABLE_16BIT
+#define POW2COEFF_16BIT
+#define LDCOEFF_16BIT
+#define WINDOWTABLE_16BIT
+
+#elif defined(__s390x__)
+#define ARCH_PREFER_MULT_32x32
+#define ARCH_PREFER_MULT_32x16
+#define SINETABLE_16BIT
+#define POW2COEFF_16BIT
+#define LDCOEFF_16BIT
+#define WINDOWTABLE_16BIT
+
+#elif defined(__sparc__)
```

Comment:
Isn't this the exact same condition as above? Same thing for s390x below.

Reply:
Sorry, this came in twice somehow. I'll fix it in a minute...